packages/tradershub/src/features/cfd/modals/ButtonGroups/AddAccountButtonsGroup.tsx

```diff
@@ -12,12 +12,12 @@ type TAddAccountButtonsGroupProps = {
 const AddAccountButtonsGroup = ({ password }: TAddAccountButtonsGroupProps) => {
     const { show } = Provider.useModal();
     const { getCFDState } = Provider.useCFDContext();
-    const platform = getCFDState('platform') ?? PlatformDetails.mt5.platform;
+    const platform = getCFDState('platform') ?? PlatformDetails.dxtrade.platform;
```

Comment:
Why you changed it ?
Is it common for mt5 and dxtrade?
If yes then what will happen for mt5